patient/README.md

```diff
@@ -0,0 +1,75 @@
+# Patient Mapping
+
+## JSON FHIR > XML HL7
+
+A Patient resource is mapped from a HL7 Patient.
+
+| Mapped to (JSON FHIR Immunization field)        | Mapped from (XML HL7 / other source)                                                              |
+|-------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| id                                              | set value = Generated GUID                                                                        |
+| meta.profile\[0]                                | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Patient-1"`.         |
+| meta.versionId                                  | fixed value = `1521806400000`                                                                     |
+| indentifier\[0].system                          | fixed value = `https://fhir.nhs.uk/Id/nhs-number`                                                 |
+| indentifier\[0].value                           | `Patient / id [@extension]`                                                                       |
+
+
+## Example JSON
+
+<details>
+    <summary>Example JSON</summary>
+
+```
+{
+    "resource": {
+        "resourceType": "Patient",
+        "id": "82c11ce2-7d9f-49a7-930b-0a195e1e9775",
+        "meta": {
+            "versionId": "1521806400000",
+            "profile": [
+                "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Patient-1"
+            ]
+        },
+        "identifier": [
+            {
+                "system": "https://fhir.nhs.uk/Id/nhs-number",
+                "value": "5538824210"
+            }
+        ],
+    }
+}
+```
+</details>
+

```

Comment:
I think we need an unmapped fields section here as there seems to be fields we are not currently mapping, such as `patient.name`, `patient.gender` etc ...